src/main/java/de/sakpaas/backend/util/TokenUtils.java

```diff
@@ -0,0 +1,15 @@
+package de.sakpaas.backend.util;
+
+public class TokenUtils {
+
+  /**
+   * Extracts the Token from the Auth-Header.
+   *
+   * @param header The Authorization-Header
+   * @return The JWT
+   */
+  public static String getTokenFromHeader(String header) {
+    String[] splitted = header.split(" ");
+    return splitted[1];
```

Comment:
Hier müssen wir uns entscheiden, ob wir bei einem fehlerhaftem Header eine `ArrayIndexOutOfBoundsException` haben wollen oder das vorher checken und eine `IllegalArgumentException` werfen wollen, wo wir mehr "verbose" sind und in die Message rein schreiben, dass der `header` folgendes Format haben muss: "Bearer xxxxxx"

Reply:
Also ich bin für die Variante die mehr "verbose" ist (also `IllegalArgumentException`), weil der Client ja die Fehlernachricht durch gereicht bekommt. Aber das musst du entscheiden.

Reply:
Da Spring Security schon beim Aufruf prüft, ob der Header stimmt. Habe ich keine Checks mehr eingebaut. Der Client bekommt bei einem ungültigem Header sowieso ein 401.